src/tasks/AutoHeistTask.py

```
@@ -301,12 +444,17 @@ def run_path(self):
     def ensure_in_team(self):
         self.wait_until(self.is_in_team, pre_action=lambda: self.send_key("esc", interval=2))
 
-    def check_current_floor(self, floor=1):
-        """检查是否在指定楼层"""
-        floor_str = "LG" + str(floor)
-        ret = self.wait_ocr(0.04, 0.235, 0.11, 0.275, match=re.compile("LG.*"), time_out=10)
-        if ret and floor_str in ret[0].name:
-            return
+    # 检查是否在指定楼层（增加兼容性？）
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)